ui/tests/acceptance/stepDefinitions/helloContext.js

```diff
@@ -0,0 +1,16 @@
+const assert = require('assert')
+const { client } = require('nightwatch-api')
+const { Given, When, Then } = require('cucumber')
+
+Given('the user browses to the hello page', function () {
+  return client.page.helloPage().navigateAndWaitTillLoaded()
+}) 
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Trailing spaces not allowed.](https://app.codacy.com/gh/owncloud/ocis-hello/pullRequest?prid=5709816)